### ISSUE TITLE
Overflow Menu Accessibility Errors

### ISSUE THREAD

**Issue by petergramenides** [carbon-design-system/carbon#8100]
## Environment

> Operating system
Mac OS Catalina 10.15.6

> Browser
Google Chrome

> Automated testing tool and ruleset
IBM Equal Access Accessibility Checker

## Detailed description

> What version of the Carbon Design System are you using?
The latest version.

> What did you expect to happen?
I scanned the user interface I was working on with the IBM Equal Access Accessibility Checker and was expecting there not to be errors.

> What happened instead?
I spent a lot of time trying to isolate the issue and eventually I narrowed it down to the Overflow Menu. When running the Overflow Menu by itself with nothing else on the page you get accessibility violations. I tried this on the Carbon website as well and got the same result. Please see the screenshot below for more information. By default, the overflow menu component in Carbon is a type of button with sections inside so it lacks a landmark element.

> What WCAG 2.1 checkpoint does the issue violate?
<img width="1779" alt="1c6ed200-7da6-11eb-9b42-ea9dc39245e5" src="https://user-images.githubusercontent.com/30732292/111326609-0fbe6980-8643-11eb-9115-471a1fd4c24b.png">

## Steps to reproduce the issue

1. Go to the [Carbon website](https://www.carbondesignsystem.com/components/overflow-menu/code/) and click on the Overflow Menu to expand it.
2. Run the [IBM Equal Access Accessibility Checker](https://chrome.google.com/webstore/detail/ibm-equal-access-accessib/lkcagbfjnkomcinoddgooolagloogehp?hl=en-US).
3. You will see accessibility violations.

## Additional information

Thank you for taking the time to review this issue! I am a Software Engineer with Watson AIOps and noticed this issue while developing a user interface.

**Comment by emyarod** [comment]
this should resolve the issue https://github.com/carbon-design-system/carbon/issues/8078#issuecomment-797664991

just as the violation details state, the solution for this is to have a `<main>` element enclosing the content of your page or placing `role="main"` on the container element. for example in the Carbon storybook, the component is contained in a `<div role="main">` element to avoid this violation

**Comment by andreancardona** [comment]
Closing this as it seems to be resolved by @emyarod 's comment :)
